controllers/limitador_controller_test.go

```diff
@@ -196,5 +227,51 @@ var _ = Describe("Limitador controller", func() {
 				return correctReplicas && correctImage
 			}, timeout, interval).Should(BeTrue())
 		})
+		It("Should modify the ConfigMap accordingly", func() {
+			updatedLimitador := limitadorv1alpha1.Limitador{}
+			Eventually(func() bool {
+				err := k8sClient.Get(
+					context.TODO(),
+					types.NamespacedName{
+						Namespace: LimitadorNamespace,
+						Name:      limitadorObj.Name,
+					},
+					&updatedLimitador)
+
+				return err == nil
+			}, timeout, interval).Should(BeTrue())
+
+			limits := []limitadorv1alpha1.RateLimit{
+				{
+					Conditions: []string{"req.method == GET"},
+					MaxValue:   100,
+					Namespace:  "test-namespace",
+					Seconds:    60,
+					Variables:  []string{"user_id"},
+				},
+			}
+			updatedLimitador.Spec.Limits = limits
+
+			Expect(k8sClient.Update(context.TODO(), &updatedLimitador)).Should(Succeed())
+			updatedLimitadorConfigMap := v1.ConfigMap{}
+			Eventually(func() bool {
+				err := k8sClient.Get(
+					context.TODO(),
+					types.NamespacedName{
+						Namespace: LimitadorNamespace,
+						Name:      limitador.LimitsCMNamePrefix + limitadorObj.Name,
+					},
+					&updatedLimitadorConfigMap)
+
+				if err != nil {
+					return false
+				}
+
+				correctHash := updatedLimitadorConfigMap.Data[limitador.LimitadorCMHash] == "69b3eab828208274d4200aedc6fd8b19"
```

Comment:
What about doing the assertion outside of the `Eventually`? As I see, the "eventually" thing is *here* is for fetching a resource that eventually will be available, but the content should be what it should be as soon as the object exists.